wee_slack.py

```diff
@@ -2990,9 +2999,14 @@ def create_user_status_string(profile):
 def create_reaction_string(reactions):
     count = 0
     if not isinstance(reactions, list):
-        reaction_string = " [{}]".format(reactions)
+        reaction_string = " {}[{}{}]{}".format(
+                w.color("245"),
+                reactions,
+                w.color("245"),
```

Comment:
Why do you repeat the color? Can the reaction string include colors?

Reply:
I do change the emoji colors, but you’re right it’s not relevant in this commit, I removed the useless mentions